src/util/streamable_generator.h

```diff
@@ -80,6 +82,7 @@ class stream_generator_promise {
     if (isBufferLargeEnough()) {
       _value.clear();
     }
+    // _filterStream uses _value as its final backing buffer
```

Comment:
`// _filterStream appends its result to _value`

("final backing buffer" is a somewhat confusing term)